samples/local_attestation/gen_pubkey_header.sh

```diff
@@ -18,6 +18,8 @@ EOF
 printf 'static const char OTHER_ENCLAVE_PUBLIC_KEY[] =' >> "$destfile"
 while IFS="" read -r p || [ -n "$p" ]
 do
+    CR=$(printf "\r")
```

Comment:
This definitely deserves a comment to explain that it is stripping out carriage return characters generated by OpenSSL on Windows, for compatibility with sample code that expects Linux-style LF-only end-of-lines.

Also in remote_attestation version of this script.

Reply:
Yes, this deserves a comment definitely. Without this change, this script produces an unusable header file.